src/BizHawk.Common/Extensions/StringExtensions.cs

```diff
@@ -165,5 +166,9 @@ public static byte[] ToCharCodepointArray(this string str)
 		/// <remarks><c>"abc,def,ghi".TransformFields(',', s => s.Reverse()) == "cba,fed,ihg"</c></remarks>
 		public static string TransformFields(this string str, char delimiter, Func<string, string> transform)
 			=> string.Join(delimiter.ToString(), str.Split(delimiter).Select(transform));
+
+		public static bool StartsWithOrdinal(this string str, string value) => str.StartsWith(value, StringComparison.Ordinal);
```

Comment:
This specific function could be overlooked (I'd appreciate an analyzer rule for that in case you agree with this being here), but the general warning will still display due to [CA1310](https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/quality-rules/ca1310).